### PR TITLE
Add ts_attcpip.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: Jenkinsfile.broker [hunk 1]
```diff
@@ -44,11 +44,11 @@ pipeline {
...
                         jobs = ['IntegrationTests']
                     } else {
-                        jobs = ['EssCommon', 'AstroskyModel', 'ATAOS', 'ATDome', 'ATDomeTrajectory', 'ATHexapod', 'ATMCSsimulator', 'ATPneumaticssimulator', 'ATSpectrograph', 'ATWhiteLight', 
-                            'CBP', 'CRIOPy', 'Cycle', 'DateLoc', 'ddsconfig', 'DIMM', 'DREAM', 'DSM', 'EAS', 'Electrometer', 'ExternalScripts', 'FiberSpectrograph', 'GIS', 'GenericCamera', 
-                            'hexrotcomm', 'HVAC', 'IntegrationTests', 'LaserTracker', 'LinearStage', 'LOVE_commander', 'LOVE_producer', 
-                            'M2Com', 'M2simulator', 'MTAirCompressor', 'MTDome', 'MTDomeTrajectory', 'MTEEC', 'MTHexapod', 'MTMount', 'ObservatoryModel', 'PMD', 'PreCommitConfig', 
-                            'salkafka', 'Scheduler', 'SimActuators', 'StandardScripts', 'TcpIP', 'TunableLaser', 'Utils', 'Vanward', 'Watcher', 'WeatherForecast', 
+                        jobs = ['EssCommon', 'AstroskyModel', 'ATAOS', 'ATDome', 'ATDomeTrajectory', 'ATHexapod', 'ATMCSsimulator', 'ATPneumaticssimulator', 'ATSpectrograph', 'attcpip', ''ATWhiteLight',
```
Comment (teweitsai) [inline Jenkinsfile.broker]: ''ATWhiteLight' -> 'ATWhiteLight'

**Review by teweitsai — APPROVED** [review]
Looks good. Do you need to update the version history file?

File: Jenkinsfile.broker (hunk 1 above)
Comment (wvreeven) [reply Jenkinsfile.broker]: Thanks for catching that!

**Comment by wvreeven** [comment]
It doesn't look like this kind of change requires an update to version history.

**Pull request merged**